cookbook/testing/simulating_authentication.rst

```diff
@@ -45,7 +45,7 @@ with a request. The following example demonstrates this technique::
         {
             $session = $this->client->getContainer()->get('session');
 
-            $firewall = 'secured_area';
+            $firewall = 'secured_area'; // put here firewalls context if defined
```

Comment:
Maybe we can expand this a bit and put a comment like the following above this line:

```php
// the name of the firewall to authenticate too (replace this with the value
// of the "context" option if it was used in the firewall's configuration)
```
What do you think?